back/sssscs/src/main/java/com/ib/certificate/CertificateController.java

```diff
@@ -65,19 +63,17 @@ public ResponseEntity<String> makeRequest(/*@DTO(CertificateRequestCreateDto.cla
 	public ResponseEntity<String> acceptRequest(@PathVariable Long id) {
 		CertificateRequest req = certificateRequestService.findByIdAndStatus(id, Status.PENDING);
 		
-		User issuee = userService.findById(1L); // TODO: Fetch user ID from token.
-		if (issuee == null) {
-			return new ResponseEntity<>("User does not exist", HttpStatus.NOT_FOUND);
+		if (req.getParent() != null) {
+			// TODO: Get user from JWT.
+			User user = req.getParent().getOwner();
+			
+			List<CertificateRequest> requestsUserCanSign = certificateRequestService.findByIssuee(user);
+			if (!requestsUserCanSign.contains(req)) {
+				// TODO: Uncomment once we retrieve real user from JWT.
+				// throw new EntityNotFoundException(CertificateRequest.class, req.getId());
```

Comment:
Shouldn't this function be called findByOwner? findByIssuee name is kinda confusing me...

Reply:
How about `findIssuedTo` or `findByRequestRecipient`? An owner of a certificate is its creator, which semantically doesn't fit well with how we use this method.

Reply:
Oh yeah, this was certRequest. When reviewing i confused it for certs for a moment.
But wait a minute, weren't our certificates issued not to users, but to any arbitrary subject? Like, this only finds the future owner, and the cert can be issued to someone who is not the future owner, right?

Reply:
We now have a distinction between a certificate subject (who the certificate is issued to) and the certificate owner (`User`, the person who made the request for that certificate). 
What `findByIssuee(r)` does is it returns the `User` who owns the certificate `c`, which is used to sign the request `r`. It's a poorly thought out name for the method, because all it does is find a `User` who can accept/reject the request.

Reply:
Maybe rename to findByFutureOwner or something? I was really confused by the issuee name

Reply:
parentCertificateOwner? Or something

Reply:
ParentCertificateOwner downplays the importance of root certificates which have no parent certificate. I would call it `findRequestsByUserWhoCanAcceptOrRejectThem()`.

Reply:
findRequestsByUserResponsibleForThem? Idk, but issuee is actually not even true, that user you find by is not the one the request is issued to (most of the time)

Reply:
[3f49d79](https://github.com/magley/sssscs/pull/5/commits/3f49d793bbdfd472926b494b4ee23d57baf145dc).